compose/config/types.py

```diff
@@ -92,6 +92,8 @@ def parse_restart_spec(restart_config):
 
 
 def serialize_restart_spec(restart_spec):
+    if not restart_spec:
+        return ''
```

Comment:
Actually it can't be `None`, because we specified `"restart": {"type": "string"},` in `compose_schema_v*.json`, if we return `None` here, the `docker-compose config` command will output an invalid restart field.

```
(develop)[realityone@rEimu ~/compose]$ bin/docker-compose -f tests/fixtures/restart/docker-compose.yml config
networks: {}
services:
  always:
    image: busybox
    restart: always
  never:
    image: busybox
    restart: 'no'
  never-with-null:
    image: busybox
    restart: null
  on-failure:
    image: busybox
    restart: on-failure
  on-failure-5:
    image: busybox
    restart: on-failure:5
version: '2.0'
volumes: {}

(develop)[realityone@rEimu ~/compose]$ bin/docker-compose -f tests/fixtures/restart/docker-compose.yml config | bin/docker-compose -f - config
ERROR: The Compose file is invalid because:
services.never-with-null.restart contains an invalid type, it should be a string
```